Assets/Scripts/UI/MainMenuController.cs

```diff
@@ -21,8 +21,9 @@ public class MainMenuController : MonoBehaviour
     private List<TabGroup> tabGroups;
     [SerializeField]
     private Selectable defaultButton;
+    public GameObject defaultMenu;
```

Comment:
Why is this one public now?

Reply:
We have the following code in GalleryMenu.cs: 
`    public void Back(InputAction.CallbackContext ctx)
    {
        mainMenuController.SwitchToMenu(mainMenuController.defaultMenu);
        mainMenuController.SelectControl(mainMenuController.GetComponentInChildren<Button>());
    }`
    
Which simply allows the menu to be closed by pressing a back button. I plan on expanding this to the other menus as well.

Reply:
Should probably have a getDefaultMenu method instead of making it public in that case

Reply:
...should probably have a **property** with private backing field, please avoid explicit get methods for this smh